docs/File-Formats.md

```diff
@@ -813,6 +814,97 @@ RET<TAB>5979<TAB>center.edu<TAB>SAMPLE_ID_3<TAB>Fusion<TAB>unknown<TAB>yes<TAB>u
+4. **Site1_Ensembl_Transcript_Id**: Ensembl transcript ID of gene 1.
+5. **Site1_Exon**: The exon rank in the provided Ensembl transcript of gene 1.
+6. **Site1_Chromosome**: Chromosome of gene 1.
+7. **Site1_Position**: Genomic position of breakpoint of gene 1.
+8. **Site1_Description**: Description of this event at site 1. This could be the location of 1st breakpoint in case of a fusion event. E.g. `Intron of ADAM9(+): 2Kb before exon 3`.
+9. **Site2_Entrez_Gene_Id**: [Entrez Gene](http://www.ncbi.nlm.nih.gov/gene) identifier of gene 2.
+10. **Site2_Hugo_Symbol**: [HUGO](http://www.genenames.org/) gene symbol of gene 2.
+11. **Site2_Ensembl_Transcript_Id**: Ensembl transcript ID of gene 2. If the transcript is not present, the canonical transcript will be retrieved from GenomeNexus.
+12. **Site2_Exon**: The exon rank in the provided Ensembl transcript of gene 2.
+13. **Site2_Chromosome**: Chromosome of gene 2.
+14. **Site2_Position**: Genomic position of breakpoint of gene 2.
+15. **Site2_Description**: Description of this event at site 2. This could be the location of 2nd breakpoint in case of a fusion event. E.g. `Intron of RNF170(-): 203bp after exon 5`.
+16. **Site2_Effect_On_Frame**: The effect on frame reading in gene 2. `Frame_Shift` or `InFrame`.
+17. **NCBI_Build**: The NCBI assembly. `GRCh37` is currently the only supported reference genome for structural variants.
+18. **DNA_Support**: Whether this observation is based on DNA. `yes`/`no`.
+19. **RNA_Support**: Whether this observation is based on RNA. `yes`/`no`.
+20. **Normal_Read_Count**: The total number of reads of the normal tissue.
+21. **Tumor_Read_Count**: The total number of reads of the tumor tissue.
+22. **Normal_Variant_Count**: The number of reads of the normal tissue that have the variant/allele.
+23. **Tumor_Variant_Count**: The number of reads of the tumor tissue that have the variant/allele.
+24. **Normal_Paired_End_Read_Count**: The number of paired-end reads of the normal tissue that support the call.
+25. **Tumor_Paired_End_Read_Count**: The number of paired-end reads of the tumor tissue that support the call.
+26. **Normal_Split_Read_Count**: The number of split reads of the normal tissue that support the call.
+27. **Tumor_Split_Read_Count**: The number of split reads of the tumor tissue that support the call.
+28. **Annotation**: Free text description of the gene or transcript rearrangement, e.g.:
+```
+PAX8 (NM_003466) rearrangement: c.1088-2028_c.25+11587del;
+TMPRSS2 (NM_001135099) - ERG (NM_182918) fusion: c.56-1943:TMPRSS2_c.18+13593:ERGdel
+```
+29. **Breakpoint_Type**: `PRECISE` or `IMPRECISE` which explain the resolution. Fill in `PRECISE` if the breakpoint resolution is known down to the exact base pair.
+30. **Center**: The sequencing center.
+31. **Connection_Type**: `3to5` or `5to3`. The direction of the second gene.
+32. **Event_Info**: Description of the event. For a fusion event, fill in `Fusion`.
```

Comment:
@sandertan is this validated? I think it should be. Alternatively, you could add the SV type to the endpoint/api as proposed in my comment of the previous review.

Reply:
Yes it's validated. I've changed the warning to error, because Fusion is currently the only supported SV type.